### PR TITLE
Make release pending wording consistent

### LINKED ISSUE #192: runningRelease resolution logic flawed
This [code](https://github.com/heroku/heroku-apps/blame/bac7045dd08faa547f122bf6d16a5e74ae3cdbc7/src/commands/releases/index.js#L37) determines this `runningRelease`:

```js
let runningRelease = releases.filter((r) => r.status === 'pending').slice(-1)[0]
```

This logic depends on how many and which other releases are in `releases`. For example, in the example below, if there are 29 releases, the last two are `pending` (in JSON), so they are displayed the user as v289 being `pending` and v289 being `release command executing`.

```term
$ heroku releases -a redacted-app -n 29
=== redacted-app Releases - Current: v316
...
v289  Set REDACTED1 config vars pending              user@salesforce.com           2017/08/24 03:20:03 +0900
v288  Set REDACTED2 config vars release command executing  user@salesforce.com           2017/08/24 03:19:52 +0900
```

However, by only fetching 28 releases, v289 changes to `release command executing` because it is now considered the `runningRelease`:

```term
$ heroku releases -a redacted-app -n 29
=== redacted-app Releases - Current: v316
...
v289  Set REDACTED1 config vars release command executing              user@salesforce.com           2017/08/24 03:20:03 +0900
```

In addition to this problem, it is questionable why the last release is considered the `runningRelease` at all. Shouldn't API be getting this information from the runtimes? I don't see how the CLI would be able to determine this on it's own.

### PR DESCRIPTION
**Pull request by dmathieu**
This was introduced to be consistent with releases not running concurrently.
That project never came to be though.

Closes #192

### CODE REVIEW

**Comment by dmathieu** [comment]
The failing test seems to be entirely unrelated to this change, and due to project-wide styling issues.

File: src/status_helper.js [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
 
-function pendingDescription (release, runningRelease, runningSlug) {
-  if (runningRelease && runningRelease.id === release.id && hasReleasePhase(runningSlug)) {
-    return 'release command executing'
-  } else {
-    return 'pending'
-  }
-}
-
 function hasReleasePhase (slug) {
```
Comment (jdxcode) [inline src/status_helper.js]: this is where the linter is failling, did you intend to use this?

**Comment by jdxcode** [comment]
not sure why the build started failing, but I fixed it and pushed up the fix to this branch for you

File: src/status_helper.js (hunk 1 above)
Comment (dmathieu) [reply src/status_helper.js]: Indeed, it's not used anymore and I missed that. I've removed it.

**Comment by dmathieu** [comment]
I saw a lot of warnings and thought it was the failure reason, missing the one error due to my change 😄

**Pull request merged**